lib/bootstrap_form/form_builder.rb

```diff
@@ -245,6 +246,32 @@ def fields_for_with_bootstrap(record_name, record_object = nil, fields_options =
 
     bootstrap_method_alias :fields_for
 
+    # Add bootstrap formatted submit button. If you need to change its type or
+    # add another css class, you need to override all css classes like so:
+    #
+    #   <%= form.submit class: "btn btn-info custom-class" %>
+    #
+    # You may add additional content that directly follows the button. Here's
+    # an example of a cancel link:
+    #
+    #   <%= form.submit do %>
+    #     <%= link_to "Cancel", "/", class: "btn btn-link" %>
+    #   <% end %>
+    #
+    def submit(value = nil, options = {}, &block)
+      out = super(value, options.reverse_merge(class: "btn"))
+      out += capture(&block) if block_given?
+
+      form_group do
+        out
+      end
+    end
+
```

Comment:
Would it make sense to define the `submit_without_bootstrap` method so the user can choose to do their own button, considering the additional functionality that this PR introduces?

Reply:
You can just do `submit_tag` to bypass builder. Same effect.